src/pages/lib-v1/auth/manageusers/q/platform/[platform].mdx

```diff
@@ -7,694 +7,103 @@ This section covers the client-side development experience for password and user
-    }
-const newPassword = 'newPassword'
-    const loggedInUser = await Auth.completeNewPassword(
-      user, // the Cognito User Object
-      newPassword, // the new password
-      // OPTIONAL, the required attributes
-      {
-        email: 'xxxx@example.com',
-        phone_number: '1234567890'
-      }
-    )
-
-    console.log(loggedInUser);
-  } catch(err) {
-    console.log(err);
-  }
-};
-```
-</Block>
-<Block name="JavaScript">
-
-```js
-import { Auth } from 'aws-amplify';
-
-async function completeNewPassword(username, password) {
-  try {
-    const user = await Auth.signIn(username, password);
-const newPassword = 'newPassword'
-    if (user.challengeName === 'NEW_PASSWORD_REQUIRED') {
-      const { requiredAttributes } = user.challengeParam; // the array of required attributes, e.g ['email', 'phone_number']
-      const loggedInUser = await Auth.completeNewPassword(
-        user, // the Cognito User Object
-        newPassword, // the new password
-        // OPTIONAL, the required attributes
-        {
-          email: 'xxxx@example.com',
-          phone_number: '1234567890'
-        }
-      )
-
-      console.log(loggedInUser);
-    }
-  } catch(err) {
-    console.log(err);
-  }
-};
-```
-</Block>
-</BlockSwitcher>
+<Fragments fragments={{js: jsResetPassword}}/>
 
-## Account recovery verification
+To complete the password reset process, invoke the `confirmResetPassword` API with the code you were sent and the new password you want to update.
 
-Either the phone number or the email address is required for account recovery. You can let the user verify those attributes by:
+import jsCompleteResetPassword from "/src/fragments/lib-v1/auth/js/password_management/confirm_reset_password.mdx"
 
-<BlockSwitcher>
-<Block name="TypeScript">
+<Fragments fragments={{js: jsCompleteResetPassword}}/>
 
-```ts
-// To initiate the process of verifying the attribute like 'phone_number' or 'email'
-async function verifyCurrentUserAttribute(attr: string) {
-  try {
-    await Auth.verifyCurrentUserAttribute(attr);
-    console.log('a verification code is sent');
-  } catch(err) {
-    console.log('failed with error', err);
-  }
-};
+### Update password
 
-// To verify attribute with the code
-async function verifyCurrentUserAttributeSubmit(attr: string, verificationCode: string) {
-  try {
-    await Auth.verifyCurrentUserAttributeSubmit(attr, verificationCode);
-    console.log('phone_number verified');
-  } catch(err) {
-    console.log('failed with error', err);
-  }
-};
-```
-</Block>
-<Block name="JavaScript">
-
-```javascript
-// To initiate the process of verifying the attribute like 'phone_number' or 'email'
-async function verifyCurrentUserAttribute(attr) {
-  try {
-    await Auth.verifyCurrentUserAttribute(attr);
-    console.log('a verification code is sent');
-  } catch(err) {
-    console.log('failed with error', err);
-  }
-};
+A signed in user can update their password using the `updatePassword` API.
```

Comment:
Nit on some of these, we're writing for the perspective of a developer writing something to support their users, not the users themselves.

```suggestion
You can update a signed in user's password using the `updatePassword` API.
```